### DISCUSSION TITLE
Showing error previews in a table

### DISCUSSION THREAD

**Discussion by ang-twilio** [twilio-labs/paste#1326] [Q&A] [answered]
My developer and I are discussing what would be an understandable and obvious way to show a preview of an error in a table. In this instance, there is an error with one of the items in the table and we want to show a preview of the error so the customer can see it at a glance and they would go into the table item to see more information about it.

One way that we see this behavior in the current Twilio console is by highlighting the error row in red and then showing the error when the customer hovers over the row. This similar paradigm exists in the message logs place. (Ignore the error text)
![image](https://user-images.githubusercontent.com/71658977/112894967-27194000-90a2-11eb-97cd-74d36c3056c3.png)

Another way we've brainstormed to solve this problem is to add an info icon next to the status and then show the error text on hover. 
![image](https://user-images.githubusercontent.com/71658977/112895270-82e3c900-90a2-11eb-9aee-782bc9f6c13c.png)

Any suggestions or guidance in how to accomplish this using the right components? Thank you!

**Comment by SiTaggart** [comment, marked as answer]
Hi @ang-twilio 

Going to cross link to another discussion related to this topic https://github.com/twilio-labs/paste/discussions/1296#discussioncomment-499700

Slightly biased, but I'm super into showing row errors at the start of the row for scalability. Something we did as Salesforce is reserve the first column to add an icon button that would show a popover when clicked, that served as error indicator and further explanation of that error. https://www.lightningdesignsystem.com/components/data-tables/#Edited-cell-with-focused-row-level-error-(unsaved)

![image](https://user-images.githubusercontent.com/368249/112915822-1245a800-90b4-11eb-9403-cb4a5f6c8fe6.png)

**Reply by ang-twilio** [reply]
Ah thank you Simon! I'll talk to my dev about incorporating something similar.
